envsec/internal/auth/user.go

```diff
@@ -17,6 +17,8 @@ var orgIDClaim = envvar.Get(
 	"org_id",
 )
 
+const legacyOrgIDClaim = "https://auth.jetpack.io/org_id"
```

Comment:
oh, do we want the org id claim of the new token be also namespaced? `https://accounts.jetpack.io/org_id`? Or do we not care

Reply:
We don't need to do it. auth0 [recommends/requires](https://auth0.com/docs/secure/tokens/json-web-tokens/create-custom-claims) it, but it's not strictly required by OIDC standard. It just makes the token larger and a bit harder to read.

Reply:
I would leave the new one as `org_id` and I'll just clean everything up once we finish migrating.